libs/csm/rest/csm_rest_test_lib.py

```diff
@@ -291,3 +291,10 @@ def get_headers(self, user_name, user_password):
                            error)
             raise CTException(
                 err.CSM_REST_VERIFICATION_FAILED, error.args[0]) from error
+
+    def search_list_of_dict(self, search_key:str, search_val:str, search_in: list):
```

Comment:
looks quite generic, do we want to add to some utils instead?

Reply:
Project specific collection util does not exists yet. Filter can be used directly as it has same function.

Reply:
Yeah this is quite generic but I wasn't sure where this will go. 
But we need this logic frequently, So We I have made function for the same instead of writing this again and again. It is easier that way

Reply:
Resolving this comment for now. If required we can move to appropriate utils